osclib/stagingapi.py

```diff
@@ -1014,14 +1024,28 @@ def package_version(self, project, package):
                 pass
         return version
 
-    def load_file_content(self, project, package, filename):
+    def guess_binary_version(self, project, package, repository='standard', arch='x86_64'):
+        """
+        Return the most probable version of the package in question
+        """
+        buildinfo = self.load_file_content(project, package, '_buildenv', binary=True, repository=repository, arch=arch)
```

Comment:
On Mon, 2015-11-30 at 01:43 -0800, Ludwig Nussel wrote:
> osc api /build/openSUSE:Factory/totest/x86_64/_product:openSUSE-
> release/openSUSE-release-20151128-1.2.src.rpm?view=fileinfo_ext

Sure - when I already know the file name... so this needs to be
preceeded by:

osc api /build/openSUSE:Factory/totest/x86_64/_product:openSUSE-release

get thruogh the file list, find the .src.rpm, then pass the other api
call...

it's an option.